rules/windows/process_creation/win_susp_adfind.yml

```diff
@@ -5,8 +5,10 @@ description: Detects the execution of a AdFind for Active Directory enumeration
 references:
     - https://social.technet.microsoft.com/wiki/contents/articles/7535.adfind-command-examples.aspx
     - https://github.com/center-for-threat-informed-defense/adversary_emulation_library/blob/master/fin6/Emulation_Plan/Phase1.md
-author: FPT.EagleEye Team
+    - https://thedfirreport.com/2020/05/08/adfind-recon/
+author: FPT.EagleEye Team, omkar72, oscd.community
 date: 2020/09/26
+modified: 2020/10/11
 tags:
     - attack.discovery
     - attack.t1016
```

Comment:
Hi @omkar72 !
Thank you for listening to my recommendations!
The rule looks good, but please adjust some more **_tags_**. I think there should be tags like this:
tags:
    - attack.discovery
    - attack.t1087.002
    - attack.t1482
    - attack.t1069.002